pkg/gds/certman/certs_test.go

```diff
@@ -276,9 +276,10 @@ func (s *certTestSuite) TestCertManagerSevenDayReissuanceReminder() {
 	require.NoError(err, "could not get hotel VASP")
 	hotelVASP = s.setupVASP(hotelVASP)
 
-	// Call the certman function at 6 days, which will send
-	// the seven day cert reissuance reminder to echoVASP.
+	// Call the certman function at 6 days and 1 day, which will send seven day
+	// cert reissuance reminders to charlieVASP and hotelVASP.
 	s.updateVaspIdentityCert(charlieVASP, 6)
+	s.updateVaspIdentityCert(hotelVASP, 1)
```

Comment:
Was this test failing? I don't understand why this line is necessary now.

Reply:
Ya, hotelVASP's cert is expired so it was triggering the `<= 7 days before expiration` case by default, but now since we have a `<= 0 days before expiration` case it was triggering that and throwing off the expected emails and causing the test to fail, so I added this line to update it's cert expiration.

Reply:
That makes complete sense, thanks.